HotelBooking.Common/Class1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace HotelBooking.Common
+{
+    public class Class1
```

Comment:
what does this class for?

Reply:
sorry about that! It's my mistake. I will delete it and push again.